block-storage-attacher/utils/config/config.go

```diff
@@ -71,7 +71,7 @@ func GetConfigStringList(envKey string, defaultConf string, logger zap.Logger) [
 }
 
 type Volume struct {
-	VolId    string `json:"iqn,omitempty"`
+	VolId    string `json:"volid,omitempty"`
```

Comment:
I think this should be `volID`. Please check and update.

Reply:
Addressed the comment

Reply:
I think its not addressed. Please check https://github.com/IBM/ibmcloud-storage-utilities/blob/8b058e6c78f20670359d600437cad511dd60d48f/block-storage-attacher/watcher/pv_watcher.go#L29

You need to match the case as well.